.github/workflows/testing.yml

```diff
@@ -37,8 +37,6 @@ jobs:
   # Run the main gRPC-Go tests.
   tests:
     # Proto checks are run in the above job.
```

Comment:
This comment should be moved with the env lines and updated to something like `in the vet-proto job`.

Reply:
make sense, I can do that